src/Tickets/Commerce/Legacy_Compat.php

```diff
@@ -26,7 +26,43 @@ class Legacy_Compat extends tad_DI52_ServiceProvider {
 	 * @since 5.1.6
 	 */
 	public function register() {
+//		$this->add_actions();
+		$this->add_filters();
+	}
+
+	/**
+	 * Adds the actions required to handle legacy compatibility.
+	 *
+	 * @since TBD
+	 */
+	protected function add_actions() {
 
 	}
 
+	/**
+	 * Adds the filters required to handle legacy compatibility.
+	 *
+	 * @since TBD
+	 */
+	protected function add_filters() {
+
+		add_filter( 'tribe_events_tickets_module_name', [ $this, 'set_legacy_module_name' ] );
```

Comment:
remove extra line break
```suggestion
		add_filter( 'tribe_events_tickets_module_name', [ $this, 'set_legacy_module_name' ] );
```